R/tax_unique.R

```diff
@@ -16,12 +16,10 @@
 #' containing the species-level identifications (i.e. the specific epithet).
 #' @param genus \code{character}. The name of the column in the dataframe
 #' containing the genus-level identifications.
-#' @param family \code{character}. The name of the column in the dataframe
-#' containing the family-level identifications.
-#' @param order \code{character}. The name of the column in the dataframe
-#' containing the order-level identifications.
-#' @param class \code{character}. The name of the column in the dataframe
-#' containing the class-level identifications.
+#' @param ... \code{character}. Other named arguments specifying the names of
+#' columns of higher levels of taxonomy (e.g. subfamily, order, superclass).
+#' These arguments should be in ascending order from lowest to highest (see
```

Comment:
```suggestion
#' These arguments should be in ascending order from lowest to highest taxonomic rank (see
```
Leave it or take it, up to you!